### PR TITLE
Replace references to serial interval with generation interval

### LINKED ISSUE #109: generation vs. serial interval
We may have discussed this before...at the moment we refer to the serial interval in the package where really we mean the generation interval (as we have no notion of sypmtom onsets). I'd suggest to change the notation to use the generation interval everywhere instead.

### PR DESCRIPTION
**Pull request by jamesmbaazam**
This PR closes #109 by:

- Renaming the `serials_dist` argument to `gen_interval`
- Revising previous wordings referring to the serial interval to use "generation interval" instead.

### CODE REVIEW

**Review by sbfnk — CHANGES_REQUESTED** [review]
Yes!

I would prefer `generation_interval` to `gen_interval`. Do you think it's too much to type?

Or as a compromise we could call it `generation_time` (only 3 characters more than `gen_interval`...) which would have the benefit of being the same as in EpiNow2.

**Comment by jamesmbaazam** [comment]
> Or as a compromise we could call it generation_time (only 3 characters more than gen_interval...) which would have the benefit of being the same as in EpiNow2.

I'm a big fun of {EpiNow2}, so I'll go with this. I think there is a similar notion in {epiparameter}, so +1. 

Will make the change.

File: vignettes/projecting_incidence.Rmd [hunk 1]
```diff
@@ -92,23 +92,29 @@ t0 <- rep(days_since_index, seed_cases$cases)
...
+$\mu = 4.7$ and $\sigma = 2.9$ [@pearson2020]. The log-normal distribution is
+commonly used in epidemiology to characterise quantities such as the serial
+interval because it has a large variance and can only be positive-valued
+[@nishiura2007; @limpert2001].
+
+Note that when the distribution is described this way, it means $\mu$ and
+$\sigma$ are the expected value and standard deviation of the natural
+logarithm of the serial interval. Hence, in order to sample the
+"back-transformed" measured serial interval with expectation/mean, $E[S]$
+and standard deviation, $SD [S]$, we can use the following parametrisation:
```
Comment (jamesmbaazam) [inline vignettes/projecting_incidence.Rmd:117]: @sbfnk Could you take a quick look at this rewording for me? Thanks.

File: vignettes/epichains.Rmd [hunk 2]
```diff
@@ -237,19 +237,20 @@ simulate_summary_eg
 
 `simulate_tree_from_pop()` simulates outbreaks based on a specified population size and pre-existing immunity until the susceptible pool runs out.
   
-Here is a quick example where we simulate an outbreak in a population of size $1000$. We assume individuals have a poisson offspring distribution with mean, $\text{lambda} = 1$, and serial interval of $3$:
+Here is a quick example where we simulate an outbreak in a population of size $1000$. We assume individuals have a poisson offspring distribution with mean, $\text{lambda} = 1$, and generation time of $3$:
```
Comment (sbfnk) [inline vignettes/epichains.Rmd:240]:
```suggestion
Here is a quick example where we simulate an outbreak in a population of size $1000$. We assume individuals have a poisson offspring distribution with mean, $\text{lambda} = 1$, and fixed generation time of $3$:
```

File: vignettes/projecting_incidence.Rmd [hunk 3]
```diff
@@ -92,23 +92,29 @@ t0 <- rep(days_since_index, seed_cases$cases)
...
-quantities such as the serial interval because it has a large variance 
-and can only be positive-valued [@nishiura2007; @limpert2001]. 
+In epidemiology, the generation interval is the duration between successive
+infectious events in a chain of transmission. Similarly, the serial
+interval is the duration between observed symptom onset times between
+successive cases in a transmission chain. The generation interval is
+often hard to observe because exact times of infection are hard to
+measure hence, the serial interval is often used instead. Here, we
+use the serial interval to represent what would normally be called the
+generation interval, that is, the time between successive cases.
```
Comment (sbfnk) [inline vignettes/projecting_incidence.Rmd:104]:
```suggestion
use the serial interval and interpret the simulated case data to represent
symptom onset.
```

I wasn't quite sure what you meant so attempted to rephrase - does this make sense?

**Review by sbfnk — COMMENTED** [review]
All looking good - we're now using both generation "time" and "interval". Perhaps that's fine as they're used interchangeably in the literature but I was wondering if it would be worth doing a search/replace "generation interval" -> "generation time" everywhere and clarify in just one point that they're the same (I made a suggestion where).

File: vignettes/epichains.Rmd (hunk 2 above)
Comment (jamesmbaazam) [reply vignettes/epichains.Rmd:240]: Thanks.

**Pull request merged**